docs/Setup/Sign-in with SSB.md

```diff
@@ -4,29 +4,27 @@ To access the [WWW dashboard interface](Web%20Dashboard.md), [internal users](..
+An [internal user](../Stakeholders/Internal%20user.md) known by its SSB ID `cid` is connected to the room via secret-handshake and muxrpc. A browser client is supposedly the same person or agent as the internal user and wishes to gain access to the web dashboard. All HTTP requests MUST be done with HTTPS.
 
 The three sides (browser client, SSB peer, and room server) perform the following [challenge-response authentication](https://en.wikipedia.org/wiki/Challenge%E2%80%93response_authentication) protocol, specified as a UML sequence diagram. We use the shorthands `cc`, `sr`, `sc`, and `cr` to mean:
 
 - `cc`: "client's challenge"
-- `sr`: "server's response"
 - `sc`: "server's challenge"
 - `cr`: "client's response"
 
-The challenges, `cc` and `sc`, are 256-bit [cryptographic nonces](https://en.wikipedia.org/wiki/Cryptographic_nonce) encoded in base64. The responses, `sr` and `cr`, are cryptographic signatures using the cryptographic keypairs that identify the server and the client, respectively, described below:
+The challenges, `cc` and `sc`, are 256-bit [cryptographic nonces](https://en.wikipedia.org/wiki/Cryptographic_nonce) encoded in base64. The response `cr` is a cryptographic signature using the cryptographic keypair that identifies the client, described below:
 
 - `cid` is the client's identity from their cryptographic keypair
 - `sid` is the servers's identity from their cryptographic keypair
 - `cc` is a 256-bit nonce created by the client, encoded in base64
-- `sc` is a 256-bit nonce created by the client, encoded in base64
-- `sr` is the server's cryptographic signature of the string `=http-auth-sign-in:${cid}:${sid}:${cc}:${sc}` where `${x}` means string interpolation of the value `x`
+- `sc` is a 256-bit nonce created by the server, encoded in base64
 - `cr` is the client's cryptographic signature of the string `=http-auth-sign-in:${cid}:${sid}:${cc}:${sc}` where `${x}` means string interpolation of the value `x`
 
 Both sides generate the nonces, but there are use cases where one side should start first. In other words, the challenge-response protocol described here can be either **client-initiated** or **server-initiated**.
 
 #### Client-initiated protocol
 
-In the client-initiated variant of the challenge-response protocol, the first step is the client creating `cc` and opening a web page in the browser. Then, the server attending to that HTTP request will call `httpAuth.signIn(cc, sc, sr)` on the client SSB peer.
+In the client-initiated variant of the challenge-response protocol, the first step is the client creating `cc` and opening a web page in the browser. Then, the server attending to that HTTP request will call `httpAuth.signIn(cc, sc)` on the client SSB peer.
```

Comment:
This is actually `httpAuth.signIn(sc, cc, cr)` but no worries, I'll correct this in PR #31

Reply:
`cr` is created by the peer, which creates it and passes it to the server as the return value of the call, no? The server can't have it yet at that point.